python/CONTRIBUTING.rst

```diff
@@ -0,0 +1,68 @@
+Bulding the Python Cookbook
+===========================
+
+The python cookbook is based on Sphinx documentation system.
```

Comment:
is based on Sphinx documentation system -> uses the Sphinx documentation system ?